fpdf/line_break.py

```diff
@@ -169,7 +169,9 @@ def get_width(
         """
 
         if chars is None:
-            chars = self.characters[start:end]
+            chars = self.characters[start:end]  # type: ignore[assignment]
+        assert chars is not None  # make mypy happy
```

Comment:
self.characters being empty should not normally happen, but it might be reasonable to just return 0.0 as the width if it does.